action.yml

```diff
@@ -1,5 +1,6 @@
-name: 'Setup ROS2 environment'
-description: 'Install ROS 2 from source on the host machine'
+name: 'Setup ROS environment'
+description: |
```

Comment:
Why do you need a pipe for a description?

Reply:
https://stackoverflow.com/questions/15540635/what-is-the-use-of-the-pipe-symbol-in-yaml